xLights/effects/PicturesEffect.cpp

```diff
@@ -655,10 +660,12 @@ void PicturesEffect::Render(RenderBuffer& buffer,
 
             if (loopGIF) {
                 image = gifImage->GetFrameForTime((buffer.curPeriod - buffer.curEffStartPer) * buffer.frameTimeInMs * frameRateAdj, true);
+                image = ApplyOrientation(image, cache->orientation);
             }
             else {
                 int ii = cache->imageCount * buffer.GetEffectTimeIntervalPosition(frameRateAdj) * 0.99;
                 image = gifImage->GetFrame(ii);
+                image = ApplyOrientation(image, cache->orientation);
```

Comment:
does the gif file format support this rotation meta data? maybe just wasting CPU cycles checking it on GIF frames